From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] vendor_lineage: drop charger animation for GSI

lineage_charger_animation menginstal battery_fail.png ke
product/etc/res/images/charger, bertabrakan dengan modul charger
milik device/phh/treble sehingga kati gagal:

  overriding commands for target
  `.../system/product/etc/res/images/charger/battery_fail.png'

Menonaktifkan lewat WITH_LINEAGE_CHARGER / PRODUCT_PACKAGES -= dari
sisi device terbukti tidak berpengaruh (rantai inherit mengevaluasi
blok ini lebih dulu dengan cara yang tidak bisa di-override dari
luar), jadi buang langsung di sini. Charger bawaan treble sudah cukup
untuk GSI. charger_res_images (AOSP) dibiarkan apa adanya.

---
 config/common_mobile.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/config/common_mobile.mk b/config/common_mobile.mk
index 0000000..0000000 100644
--- a/config/common_mobile.mk
+++ b/config/common_mobile.mk
@@ -36,12 +36,6 @@ PRODUCT_PACKAGES += \
 # Charger
 PRODUCT_PACKAGES += \
     charger_res_images

-ifneq ($(WITH_LINEAGE_CHARGER),false)
-PRODUCT_PACKAGES += \
-    lineage_charger_animation \
-    lineage_charger_animation_vendor
-endif
-
 # Credential storage
 PRODUCT_PACKAGES += \
     android.software.credentials.prebuilt.xml
--
2.54.0
